README.md

```diff
@@ -805,6 +806,38 @@ data, err := fgaClient.WriteAssertions(context.Background()).
 ```
 
 
+### Retries
+
+If a network request fails with a 429 or 5xx error from the server, the SDK will automatically retry the request up to 15 times with a minimum wait time of 100 milliseconds between each attempt.
+
+To customize this behavior, create an `openfga.RetryParams` struct and assign values to the `MaxRetry` and `MinWaitInMs` fields. `MaxRetry` determines the maximum number of retries (up to 15), while `MinWaitInMs` sets the minimum wait time between retries in milliseconds.
+
+Apply your custom retry values by passing this struct to the `ClientConfiguration` struct's `RetryParams` parameter.
+
+```golang
+import (
+	"os"
+
+	openfga "github.com/openfga/go-sdk"
+	. "github.com/openfga/go-sdk/client"
+)
+
+func main() {
+	fgaClient, err := NewSdkClient(&ClientConfiguration{
+		ApiUrl:               os.Getenv("FGA_API_URL"),                // required, e.g. https://api.fga.example
+		StoreId:              os.Getenv("FGA_STORE_ID"),               // not needed when calling `CreateStore` or `ListStores`
+		AuthorizationModelId: os.Getenv("FGA_AUTHORIZATION_MODEL_ID"), // optional, recommended to be set for production
```

Comment:
Sorry to ask, but considering your updating all the READMEs, do you mind changing this in all to be `FGA_MODEL_ID`? In order to be consistent with the CLI

Reply:
Sure thing, will update all references of `FGA_AUTHORIZATION_MODEL_ID` in the readme's and also any other code that references it (like examples)